### ISSUE TITLE
cuDNN launch failure

### ISSUE THREAD

**Issue by kannyjyk** [kimmo1019/Roundtrip#4]
I'm running the codes using Tensorflow 1.13.1 (Python 2.7) on GPU in a conda environment. And I get the error:

> InternalError (see above for traceback): cuDNN launch failure : input shape ([64,128,1,1])
> 	 [[node dx_net/BatchNorm/FusedBatchNorm (defined at /public/home/***/Roundtrip/model.py:37) ]]
> 	 [[node Mean_4 (defined at main_density_est.py:68) ]]

I can't quite figure out why it happened, I also tried the methods mentioned on the websites (add `os.environ['TF_FORCE_GPU_ALLOW_GROWTH'] = 'true'` or change the batch size) but it did not change anything. So do you have any suggestions?

Thank you!

**Comment by kimmo1019** [comment]
> I'm running the codes using Tensorflow 1.13.1 (Python 2.7) on GPU in a conda environment. And I get the error:
> 
> > InternalError (see above for traceback): cuDNN launch failure : input shape ([64,128,1,1])
> > [[node dx_net/BatchNorm/FusedBatchNorm (defined at /public/home/***/Roundtrip/model.py:37) ]]
> > [[node Mean_4 (defined at main_density_est.py:68) ]]
> 
> I can't quite figure out why it happened, I also tried the methods mentioned on the websites (add `os.environ['TF_FORCE_GPU_ALLOW_GROWTH'] = 'true'` or change the batch size) but it did not change anything. So do you have any suggestions?
> 
> Thank you!

I guess it's the cuda or cudnn environment problem, I'm using CUDA10.0 and cudnn7.6. You can refer to https://www.tensorflow.org/install/source#gpu to help you find a suitable CUDA and cudnn version. 
The cudnn should be located in /path/to/cuda/lib64, also make sure this path is added to $LD_LIBRARY_PATH.

**Comment by kannyjyk** [comment]
I solved it by using CUDA10.0 and cudnn7.4 as https://www.tensorflow.org/install/source#gpu. Thanks!!
